### PR TITLE
Add tooltip to session recording, kill feature flag

### PR DESCRIPTION
**Pull request by macobo**
## Changes

*Please describe.*  
*If this affects the front-end, include screenshots.*  

## Checklist

- [ ] All querysets/queries filter by Organization, Team, and User (if this PR affects ANY querysets/queries).
- [ ] Django backend tests (if this PR affects the backend).
- [ ] Cypress end-to-end tests (if this PR affects the frontend).

### CODE REVIEW

File: frontend/src/scenes/sessions/SessionsTable.tsx [hunk 1]
```diff
@@ -98,17 +95,23 @@ export function SessionsTable({ personIds, isPersonPage = false }: SessionsTable
...
         },
-    ]
-
-    if (!user?.is_multi_tenancy || featureFlags['session-recording-player']) {
-        columns.push({
-            title: 'Play Session',
+        {
+            title: (
+                <span>
+                    <Tooltip title="Enable session recording from Project > Settings">
```
Comment (paolodamico) [inline frontend/src/scenes/sessions/SessionsTable.tsx:101]: Maybe add a direct link to the settings page from here?

File: frontend/src/scenes/project/Settings/index.js [hunk 2]
```diff
@@ -97,26 +95,19 @@ function _Setup() {
...
-                            </a>{' '}
-                            with us!
-                        </p>
-                    </>
-                )}
+                <h2 id="sessionrecording" className="subtitle">
+                    Session recording <span style={{ fontSize: 16, color: 'var(--warning)' }}>BETA</span>
+                </h2>
+                <p>Watch sessions replays to see how users interact with your app and find out what can be improved.</p>
+                <OptInSessionRecording />
```
Comment (paolodamico) [inline frontend/src/scenes/project/Settings/index.js:102]: Unrelated to this PR but it'd be great if we show a toast when the setting is saved (i.e. there's no feedback right now if the setting was properly updated)

**Review by paolodamico — APPROVED** [review]
some improvement recommendations, but lgtm!

File: frontend/src/scenes/sessions/SessionsTable.tsx (hunk 1 above)
Comment (macobo) [reply frontend/src/scenes/sessions/SessionsTable.tsx:101]: Not sure what you have in mind design-wise - mind elaborating?

Comment (paolodamico) [reply frontend/src/scenes/sessions/SessionsTable.tsx:101]: Thinking just turn "Settings" into a link? or even if you click the tooltip body?

**Pull request merged**